python/sidecar/http.py

```diff
@@ -230,10 +226,10 @@ async def delete_project(project_id: str):
 
 
 @project_api.get("/{project_id}/files")
-async def get_project_files(project_id: str):
+async def get_project_files(project_id: str) -> list[FileEntry | FolderEntry]:
```

Comment:
nit: I would prefer to have a type (ex: `ProjectFiles`) that would contain a filed with this list. Returning arrays in the output prevents the output from evolving.